### PR TITLE
NISP-2297: NIRecord and Liabilities caching code and test

### PR DESCRIPTION
**Pull request by prashantchoudhari**

### CODE REVIEW

**Comment by prashantchoudhari** [comment]
@dspork - Please review.

**Comment by prashantchoudhari** [comment]
@swaistle - 
1. Please test this branch (acceptance and API). 
2. Change flag HOD one to true and false

File: test/uk/gov/hmrc/nationalinsurancerecord/cache/StubApplicationConfig.scala [hunk 1]
```diff
@@ -19,5 +19,5 @@ package uk.gov.hmrc.nationalinsurancerecord.cache
 import uk.gov.hmrc.nationalinsurancerecord.config.ApplicationConfig
 
 object StubApplicationConfig extends ApplicationConfig {
-  override val responseCacheTTL: Int = 60
+  override val responseCacheTTL: Int = 30
```
Comment (dspork) [inline test/uk/gov/hmrc/nationalinsurancerecord/cache/StubApplicationConfig.scala]: Is there a reason for this change?

Comment (prashantchoudhari) [inline test/uk/gov/hmrc/nationalinsurancerecord/cache/StubApplicationConfig.scala]: will change it back to 60, as Jenkins takes more time.

**Pull request merged**